README.md

```diff
@@ -16,7 +16,34 @@ http://developers.google.com/api-client-library/php
 
 ## Installation ##
 
-For the latest installation and setup instructions, see [the documentation](https://developers.google.com/api-client-library/php/start/installation).
+The [Releases](https://github.com/google/google-api-php-client/releases) page lists all stable versions. Download any file
+with the name `google-api-php-client-[RELEASE_NAME].zip` for a package including this library and its dependencies.
+
+### Composer
+
+Install this library via [composer](https://getcomposer.org) by running the following command:
+
+```sh
+# change directories to the root of your project
+cd /path/to/my-project
+# if you don't already have composer, install it
+curl -sS https://getcomposer.org/installer | php
```

Comment:
I am really trying to keep this guide simple. As the stack overflow thread shows, some users are entirely unfamiliar with composer. Sending them to the website for installation will just increase the bounce rate. That is why I went this way... to get them up and going the fastest.